src/Sarif.Viewer.VisualStudio/SLoadSarifLogService.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal interface SLoadSarifLogService
```

Comment:
>SLoadSarifLogService [](start = 23, length = 20)

Bad name? Interfaces should be prefixed with I

Reply:
Is this just an erroneous file you added that needs to be deleted?

---
In reply to: [211049173](https://github.com/Microsoft/sarif-visualstudio-extension/pull/9#discussion_r211049173) [](ancestors = 211049173)

Reply:
No, you need both. It's a kinda weird paradigm. I can't find the material I followed but here is another article that shows it: [https://www.codemag.com/article/0710092/Learning-the-Visual-Studio-SDK-with-VSSDK-Assist](https://www.codemag.com/article/0710092/Learning-the-Visual-Studio-SDK-with-VSSDK-Assist)
Scroll near the bottom to "Listing 5: C# code for service implementation" and you'll see the two interfaces.

---
In reply to: [211049444](https://github.com/Microsoft/sarif-visualstudio-extension/pull/9#discussion_r211049444) [](ancestors = 211049444,211049173)

Reply:
Pretty sure this is just a "marker" interface, used by the service locator to look up the service. Once located, the service might expose multiple "functional" interfaces.

---
In reply to: [211049444](https://github.com/Microsoft/sarif-visualstudio-extension/pull/9#discussion_r211049444) [](ancestors = 211049444,211049173)